packages/core/src/utils/transformers/media/transformMedia.ts

```diff
@@ -49,13 +55,15 @@ export const transformMedia = (
     try {
       value = getOptimizedBackgroundImageAsset(
         asset.fields.file as AssetFile,
-        Number(500),
+        width as string,
         Number(quality),
         format as (typeof SUPPORTED_IMAGE_FORMATS)[number],
-      ).url;
+      );
     } catch (error) {
       console.error('Error transforming image asset', error);
     }
+  } else {
+    return getBoundValue(entityStore, binding, []);
```

Comment:
I wonder adding early returns in the `if` and `else if` statements would improve readability.

Basically making the function body more like this psuedocode:
```ts
  if (variableName === 'cfImageAsset') {
    try {
      return getOptimizedImageAsset(...)
    } catch {
      console.error(...)
    }
  } else if (variableName === 'cfBackgroundImageUrl') {
    try {
      return getOptimizedBackgroundImageAsset(...)
    } catch {
      console.error(...)
    }
  }
  return getBoundValue(...)
  ```
  
  What are your thoughts?

Reply:
Ya usually pretty good change